test/runtests.jl

```diff
@@ -127,4 +127,15 @@ const is_TRAVIS = haskey(ENV, "TRAVIS")
             include("pde_systems/MOL_1D_Linear_Convection.jl")
         end
     end
+    ############### JSC #####################
+    if GROUP == "All" || GROUP == "MOL_Interface2_JSC"
+        @time @safetestset "MOLFiniteDifference Interface" begin
```

Comment:
Since `JuliaSimCompiler` is an extension, let's add it explicitly before testing integraton, instead of operating on the implicit assumption that it's already available to the test environment.
```suggestion
	Pkg.add("JuliaSimCompiler")
        @time @safetestset "MOLFiniteDifference Interface" begin
```